draft-irtf-cfrg-vdaf.md

```diff
@@ -1168,18 +1177,19 @@ of the `Vdaf` interface specify a method of encoding the `PublicShare`,
 
 Each VDAF is identified by a unique, 32-bit integer `ID`. Identifiers for each
 (V)DAF specified in this document are defined in {{codepoints}}. The following
-method is defined for every VDAF:
+method is defined for each VDAF specified in this document:
```

Comment:
Forgot to note this in the commit: worth clarifying that this is for Prio3 and Poplar1. Other VDAFs don't need to version the same way, and perhaps they shouldn't. They'll need to version their own drafts anyway.